### DISCUSSION TITLE
有没有可能增加移动端客户端

### DISCUSSION THREAD

**Discussion by sikenali** [SPlayer-Dev/SPlayer#105] [想法 Ideas]
RT

**Comment by imsyy** [comment]
由于 `Electron` 不支持构建移动端，所以不行，不过移动端可以使用 `Web` 端（网页版），后续会对移动端进行适配

**Comment by sumitroajiprabowo** [comment]
Hi @sunurich, thanks for the question!

**Current state:** the tool is designed as **one instance per OLT** — `SNMP_HOST` points to a single device. For your 30 OLTs, the recommended approach today is to deploy 30 Helm releases with different `snmp.host` values:

```bash
helm install snmp-olt-site-01 ./snmp-olt-zte-c320 --set snmp.host=10.0.0.1
helm install snmp-olt-site-02 ./snmp-olt-zte-c320 --set snmp.host=10.0.0.2
```

Each instance gets its own Redis cache, trap listener, and webhook config — fully isolated.

**Roadmap:** I'm planning to extend this module to support **multi-OLT management** for both **ZTE C300 and C320** in a single instance. The OID trees of C300 and C320 are largely compatible, so a unified OLT profile abstraction should work well.

Will share progress here once development starts. Stay tuned! 🚀
